### DISCUSSION TITLE
欢迎来到医学影像数据集项目讨论区

### DISCUSSION THREAD

**Discussion by linhandev** [linhandev/dataset#8] [综合]
找医学影像数据集是一个蛮麻烦的过程，希望本项目能帮助到大家。
欢迎在讨论区
- 指出项目内容错误
- 为项目提出建议
- 咨询有关数据集的任何问题

**Comment by Treving** [comment]
想请问一下下载完数据之后要怎么在代码里使用数据集，label是需要自己再在代码里定吗

**Comment by TalenWang-AIcv** [comment]
您好，邮件已收到，我会尽快处理，谢谢！
